### PR TITLE
Show last state even if connection error

### PR DESCRIPTION
**Pull request by aldipower**
According to #466 show the last fetched state for applist, tasklist, tasks details and deploymentlist, even if there is an fetch error. Shows the error message under the lists table header. Message disappears if connection is sucessfull.

### CODE REVIEW

File: src/main/resources/assets/js/components/TaskDetailComponent.jsx [hunk 1]
```diff
@@ -80,11 +86,8 @@ define([
             <li className="active">{task.get("id")}</li>
           </ol>
           <h5>Task Details</h5>
+          {fetchError}
           {
```
Comment (mlunoe) [inline src/main/resources/assets/js/components/TaskDetailComponent.jsx]: Let's remove the unnecessary brackets here and move the markup below back in line

File: src/main/resources/assets/js/components/TaskListComponent.jsx [hunk 2]
```diff
@@ -71,16 +81,7 @@ define([
...
-        taskNodes =
-          <tbody>
-            <tr>
-              <td className="text-center text-danger" colSpan="7">
-                Error fetching tasks. Refresh the list to try again.
-              </td>
-            </tr>
-          </tbody>;
-      } else if (tasksLength === 0) {
+      } else if (tasksLength === 0 && !fetchError) {
```
Comment (mlunoe) [inline src/main/resources/assets/js/components/TaskListComponent.jsx]: Can you create a variable here like it is done in TaskListDetail to tell if there's an error?
```
var hasError =
    this.props.fetchState === States.STATE_ERROR ||
    tasksLength === 0;
```

**Comment by mlunoe** [comment]
Other than that, it looks good! :)

**Comment by drexin** [comment]
:+1:

**Pull request merged**